exporters/Makefile

```diff
@@ -0,0 +1,58 @@
+PWD                   = $(shell pwd)
+NODE_EXPORTER_VERSION = $(or $(shell printenv NODE_EXPORTER_VERSION), "")
+KUBE_STATE_METRICS    = $(or $(shell printenv KUBE_STATE_METRICS), "")
+
+
+.PHONY: build
+build: kube-state-metrics prometheus-node-exporter
+
+.PHONY: kube-state-metrics
+kube-state-metrics: docker-kustomize-kube-state-metrics docker-vanilla-kube-state-metrics
+
+.PHONY: kustomize-kube-state-metrics
+kustomize-kube-state-metrics: kube-state-metrics-template kube-state-metrics-kustomization.yaml
+
+.PHONY: kube-state-metrics-template
+kube-state-metrics-template:
+	helm template kube-state-metrics stable/kube-state-metrics --version=${KUBE_STATE_METRICS} --output-dir .
+
+.PHONY: vanilla-kube-state-metrics
+vanilla-kube-state-metrics:
+	@mkdir -p vanillas
+	helm template kube-state-metrics stable/kube-state-metrics --version=${KUBE_STATE_METRICS} > vanillas/vanilla-kube-state-metrics.yaml
```

Comment:
I would make these driven by parameters, and then you can have the one target to run helm template, and you just call it from different targets. Also if you drop the `vanilla-` prefix from the name, the chart, repo name, and output would all be the same, so you could get away with only two params probably.

So as a hypothetical example the following:

```
RELEASE_CHANNEL = "stable"

.PHONY: template
template:
	helm template ${CHART} ${RELEASE_CHANNEL}/${CHART} --version=${VERSION} > vanillas/${CHART}.yaml

kube-state-metrics-template:
	make template CHART=kube-state-metrics VERSION=${KUBE_STATE_METRICS}
```